audio_synthesizer.py

```diff
@@ -119,7 +122,7 @@ def synthesize_mixtures(self):
                         if self._db_name == 'nigens':
                             eventsig, fs_db = soundfile.read(self._db_path + '/' + filename)   
                         elif self._db_name == 'fsd50k':
-                            eventsig, fs_db = soundfile.read(self._db_path + '/' + filename.split('/')[-1])
+                            eventsig, fs_db = librosa.load(filename, sr=self._fs, duration=onoffset[1]/10) # here we need librosa since we are loading .mp3 

```

Comment:
I went ahead and added the duration argument to `librosa.load`

Reply:
It looks like librosa load takes a `float` that represents seconds. From the metadata I observe we set `[0, 10]`, which I assumed to be start at second `0` and end at second `10`. If we do `duration=onoffset[1]/10`, would this be trimming up to `1s` total per track?

Reply:
the `duration` argument needs to be `(onoffset[1]-onoffset[0])-0.1`

Reply:
eventsig, fs_db = librosa.load(path=self._db_path+'/'+filename.split('/')[-1], duration=(onoffset[1]-onoffset[0])-0.1)